msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -266,19 +266,125 @@ You can build an in-meeting app for enabling meeting participants to sign docume
+* The grace period to get memory usage down after the app is cached is 1 minute.
+* You can't enable app caching if the system memory is less than 4 GB or the available free memory is less than 1 GB (512 MB on Mac).
+* Side panel is the only supported FrameContext for app caching in meetings.
+* When the app is cached, CPU usage must not exceed 5%.
+* When the app is cached, the number of SDK requests shouldn't exceed five for every 12 seconds.
+* The cache state is monitored every 12 seconds and the apps that don’t meet the requirements are removed from the cache.
+
+### Code example
+
+The following code snippet is an example to enable app caching on your app in Teams meeting:
+
+```javascript
+microsoftTeams.registerBeforeUnloadHandler((readyToUnload) => { 
+console.log("got beforeunload from TEAMS"); 
+// dispose resources and then invoke readyToUnload 
+readyToUnload(); 
+return true; 
+}); 
+
+microsoftTeams.registerOnLoadHandler((data) => { 
+console.log("got load from TEAMS", data.contentUrl, data.entityId); 
+// use contentUrl to route to correct page 
+// invoke notifySuccess when ready  
+microsoftTeams.appInitialization.notifySuccess(); 
+}); 
+```
+
+### Limitations
+
+* Single-page apps that use client-side routing for page navigation can benefit from app caching. It's recommended that the same domain is used across all contexts of your app launch. For example, using *bar.foo.com* for chats and *baz.foo.com* for personal app isn't recommended as you need to go to the new domain in the load handler.
+
+* Apps need to re-register for events such as `themeChange` and `focusEnter` in the load handler. Teams client won't send any notifications to the app when cached. If your app requires notifications even when cached, caching might not be the right solution.
+
+* App caching is supported only in Teams desktop client. In Teams web client, even if the app registers load handlers, the app is removed from the cache after the unload sequence is completed.
+
+* Register the `load` and `beforeUnload` handlers early in your launch sequence. If the Teams client doesn’t identify these registrations before the user goes out of the app, the app isn't cached.
+
+* The Teams client invokes the `loadHandler` only after the `unload` sequence of the app is completed. For example, if a user launches tab A of your app and then launches tab B of the same app, tab B won't get the load signal until the tab B invokes the `readyToUnload` callback.
+
+* After an app moves to the cached state, it has a one minute grace period to get the memory usage under the allowed threshold of 225 MB. <!-- The Memory value used for this check is the *workingSetSize* of the webview as reported by the Electron [getappMetrics](https://www.electronjs.org/docs/latest/api/app#appgetappmetrics) API. -->
+
+* Apps are cached on a per-window basis. An app cached in a meeting window can't be reused in a channel.
```

Comment:
combine this?
>>>An app cached in a meeting window can't be reused in a channel.
leave this out - APp Caching in Channels not supported at this time